template_config.yml

```diff
@@ -5,7 +5,7 @@
 
 additional_repos:
 - bindings: true
-  branch: 0.14.0
+  ref: 38b3ac1814d6d2119d17fbb2b16f0de027b52f36
```

Comment:
I was wrong. This should be `revision: 38b3ac1814d6d2119d17fbb2b16f0de027b52f36`